operationsgateway_api/src/routes/records.py

```diff
@@ -135,6 +140,7 @@ async def count_records(
 )
 @endpoint_error_handling
 async def convert_search_ranges(
+    access_token: AuthoriseToken,  # noqa: B008
```

Comment:
Is the `noqa` comment needed?

Reply:
lint isn't complaining without it so I removed it